lib/api/controllers/securityController.js

```diff
@@ -294,19 +290,23 @@ function SecurityController (kuzzle) {
     return kuzzle.pluginsManager.trigger('security:beforeGetUser', requestObject)
       .then(newRequestObject => {
         modifiedRequestObject = newRequestObject;
-
-        if (!modifiedRequestObject.data._id) {
+        if (!modifiedRequestObject.data._id && !modifiedRequestObject.data.body._id) {
```

Comment:
Isn't this test a consequence of putting the `_id` field in the body part of the request object in `authController.getCurrentUser()`?
If so, can't we just test if `modifiedRequestObject.data._id` is falsey and keep this field exclusively in the data part of a request object?